ibis/backends/base/sql/alchemy/client.py

```diff
@@ -329,3 +329,81 @@ def _sqla_table_to_expr(self, table):
     def version(self):
         vstring = '.'.join(map(str, self.con.dialect.server_version_info))
         return parse_version(vstring)
+
+    def insert(
+        self,
+        to_table_name: str,
+        database: Optional[str] = None,
+        data_obj=None,
+        from_table_name: Optional[str] = None,
+        if_exists: Optional[str] = 'append',
+        values=None,
+    ) -> None:
+        """
+        Insert the given data to a table in backend.
+
+        Parameters
+        ----------
+        to_table_name : string
+            name of the table to which data needs to be inserted
+        database : string, optional
+            name of the attached database that the table is located in.
+        data_obj : pd.Dataframe (Later can also be used for sending in
+        table_expr)
+            data_obj is the dataframe containing data which needs to be
+            inserted to to_table_name
+            (Later when table_expr related implementation is done this
+            variable can be used to take in that data)
+        from_table_name: string, optional
+            name of the table from which data needs to be inserted
+        if_exists : string, optional, default 'append'
+            The values available are: {‘fail’, ‘replace’, ‘append’}
+        values: None, optional
```

Comment:
I tried to add it as another way the user can insert data (single row) into a table similar to the `insert` method implemented in impala (`backends/impala/client.py`). This was the idea behind having a parameter for it. If you still feel it's redundant I can remove it.

Reply:
I see. But yes please, I don't think the Impala `values` is very useful, and makes things too much complex. And in any case, since we're not implementing it, let's better remove the argument.

Reply:
Sure!